msgcntr/messageforums-app/src/webapp/css/msgcntr.css

```diff
@@ -681,3 +681,12 @@ div.msgcntr-profile-qtip {
 	cursor: pointer;
 	font-size: 20px !important;
 }
+
+#revise\:closeDateSpan #revise\:closeDate, #revise\:openDateSpan #revise\:openDate {
```

Comment:
I think single ":" is now deprecated for psuedo elements in favor of double "::"?